frontend/src/modules/auth/auth-routes.js

```diff
@@ -26,19 +26,19 @@ export default [
         name: 'signup',
         path: 'signup',
         component: SignupPage,
-        meta: { unauth: true },
+        meta: { unauth: true, title: 'Signup' },
```

Comment:
Sorry just one thing I let slip before, could you update this one to `Sign up`?

Reply:
sorry for a tiny mistake. Updated it